service/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# LS docker file
+
+# build via (docker root = project root):
+# docker build -t absaoss/login-service:latest \
+# --build-arg BUILD_PROXY=http://my.cool.proxy.here:3128 \
+# --build-arg CONFIG=./path/to/my.awesome.local.application.yaml \
+# --build-arg SSL=true \
+# --build-arg SSL_DNAME="CN=*.my.domain.com, OU=project1, O=mycorp, L=Johannesburg, ST=Gauteng, C=za" .
+# run via
+# docker run -p 8080:8080 -p 8443:8443 absaoss/login-service:latest
+
+# test via:
+# http://localhost:8080/token/public-key
+# https://localhost:8443/token/public-key
+
+
+# --- Base image part (common for SSL true|false) ---
+# Conditional Docker image creation idea: https://stackoverflow.com/a/60820156/1773349
+# change to OFF to disable SSL
+ARG SSL=true
+
+# Tomcat OS base image, feel free to use another (e.g. your hardened one)
+ARG BASE_IMAGE=tomcat:9-jdk8-corretto
+FROM $BASE_IMAGE as base
+
+LABEL org.opencontainers.image.authors="ABSA"
+
+ARG BUILD_PROXY
+ARG CONFIG=./src/main/resources/example.application.yaml
+
+ENV LS_WAR_PATH=./target/scala-2.12
```

Comment:
maybe even 2.12 can be somehow parametrized, but probably good enough for now

Reply:
I am not planning on doing this presently, if in the future we support different Scala version, we can go for this option.